pkgs/by-name/vo/voms/package.nix

```
@@ -1,32 +1,33 @@
-{ lib
-, stdenv
-, fetchFromGitHub
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
   # Native build inputs
-, autoreconfHook
-, bison
-, flex
-, pkg-config
+  autoreconfHook,
+  bison,
+  flex,
+  pkg-config,
   # Build inputs
-, expat
-, gsoap
-, openssl
-, zlib
+  expat,
+  gsoap,
+  openssl,
+  zlib,
   # Configuration overridable with .override
   # If not null, the builder will
   # create a new output "etc", move "$out/etc" to "$etc/etc"
   # and symlink "$out/etc" to externalEtc.
-, externalEtc ? "/etc"
+  externalEtc ? "/etc",
 }:
 
-stdenv.mkDerivation rec{
-  pname = "voms-unstable";
-  version = "2022-06-14";
+stdenv.mkDerivation (finalAttrs: {
```

```suggestion
stdenv.mkDerivation (finalAttrs: {
```
FYI `rec` may be faster on eval than a [fixed point](https://github.com/NixOS/nixpkgs/issues/320528#issuecomment-2175087036). Changing this for the sake of changing it I don't know if it is a good idea.

---

> FYI rec may be faster on eval than a fixed point. Changing this for the sake of changing it I don't know if it is a good idea.

Glad to hear about the cons of the fixed-point arguments. Thank you for this suggestions.